constants/index.js

```diff
@@ -25,6 +25,11 @@ import {
 } from './vars'
 import { THEME, fontSize } from './theme'
 
+const SORT_DIRECTION = {
+  ASC: 'ASC',
+  DESC: 'DESC',
+}
+
```

Comment:
should we put these in vars..?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/479/commits/16b3f5de405652b1c7f1984a4499061a306f57dc